.github/workflows/MacOSbacksupport.yml

```diff
@@ -0,0 +1,75 @@
+name: Wheel build test
+on:
+  push:
+    branches:
+      - 'release/aicspylibczi'
+
+jobs:
+  wheel-setup:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macOS-10.15]
+
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          submodules: true
+      - name: Build Miniconda
+        run: |
+          wget https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh -O ~/miniconda.sh
+          bash ~/miniconda.sh -b -p $HOME/miniconda
+          export PATH=~/miniconda/bin:$PATH
+          $HOME/miniconda/bin/conda init bash
+      - name: Build 3.6
+        run: |
+          source ~/.bashrc
+          source ~/.bash_profile
+          export PATH=~/miniconda/bin:$PATH
+          $HOME/miniconda/bin/conda create -y -n jamies36 python=3.6
+          conda activate jamies36
+          export SDKROOT=/Applications/Xcode_12.3.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk
+          export MACOSX_DEPLOYMENT_TARGET=10.10
+          pip install wheel
+          python setup.py bdist_wheel
```

Comment:
There is a part of me that says these are so repeated you should have this as a make command w/ a parameter for which python target to use.

Reply:
Fair point. Part of me is really hoping that a more elegant solution emerges eventually. It strikes me as very shortsighted that the python binaries are built with minimal compatibility. (10.14)

Reply:
I created a script to clean up the action.